testdata/config/listcheck.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: "config.istio.io/v1alpha2"
+kind: listchecker
+metadata:
+  name: staticversion
+  namespace: istio-config-default
+spec:
+  # providerUrl: ordinarily black and white lists are maintained
+  # externally and fetched asynchronously using the providerUrl.
+  overrides: ["v1", "v2"]  # overrides provide a static list
+  blacklist: false
+---
+apiVersion: "config.istio.io/v1alpha2"
+kind: listentry
+metadata:
+  name: appversion
+  namespace: istio-config-default
+spec:
+  value: source.labels["version"]
+
+---
+apiVersion: "config.istio.io/v1alpha2"
+kind: rule
+metadata:
+  name: checkwl
+  namespace: istio-config-default
+spec:
+  match: destination.labels["app"] == "ratings"
+  actions:
+  - handler: staticversion.listchecker
+    instances: [ appversion.listentry ]
```

Comment:
is this style our style recommendation for instance lists?

Reply:
I have been using it, but I think we should switch to the standard way.
It is not as concise, but makes diffs easier.
I will send a PR